### PR TITLE
KeyboardAwareScrollView README

### PR DESCRIPTION
**Pull request by vshab**

### CODE REVIEW

File: packages/keyboard-aware-srollview/README.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
 # KeyboardAwareScrollView
 
-ScrollView that moves child input just above the keyboard when later gets focus
+Solves the very [common](https://www.reddit.com/r/reactnative/comments/azuy4v/were_the_react_native_team_aua/eiaa8vs/)
+React Native problem of software keyboard poping over focused text input and tries to do it accurate and smooth(even on Android).
```
Comment (mxck) [inline packages/keyboard-aware-srollview/README.md]: `smooth(even on Android).` - space.

**Review by mxck — APPROVED** [review]
Good, but check first paragraph in spellcheck. There are a few little typos.
http://www.reverso.net/spell-checker/english-spelling-grammar/

File: packages/keyboard-aware-srollview/README.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+- Smooth animation
+- Works both on iOS and Android
+- Small and simple code
+
+## Usage
+
+Just put the content with inputs in `KeyboardAwareScrollView` and that's it.
+
+List of spesific `KeyboardAwareScrollView` props and methods is availble
+[below](https://github.com/pietile/pietile-native-kit/tree/master/packages/keyboard-aware-srollview#api).
```
Comment (mxck) [inline packages/keyboard-aware-srollview/README.md]: Just #api

File: packages/keyboard-aware-srollview/README.md (hunk 1 above)
Comment (vshab) [reply packages/keyboard-aware-srollview/README.md]: What do you mean?

Comment (mxck) [reply packages/keyboard-aware-srollview/README.md]: Missing space before bracket :)

**Pull request merged**